src/int.c

```diff
@@ -23,30 +47,61 @@ void HAL_TIM_PeriodElapsedCallback(TIM_HandleTypeDef* htim)
 {
     if (htim == &htim1) {
         timer_overflows++;
+        pps_overflows++;
     } else if (htim == &htim2) {
+        // TIM2 is configure for 1 second count
+        // PPS output signal
+        HAL_GPIO_WritePin(PPS_OUTPUT_GPIO_Port, PPS_OUTPUT_Pin, 1);
```

Comment:
Is there no risk that we get jitter on the PPS output due to other interrupts? Is it possible to use PWM output on the pin instead?

Reply:
Well I would have thought so too, but it actually works pretty well!
I don't have use for PPS output, but it was a feature request from daddygo on eevblog.
He needs a PPS output that keeps steady even if the GPS signal gets lost.
Here we sync with GPS PPS output when we get it, and we can resync if we drift too much or when we reach PPB = 0 (see parameters in PPB and PPS menu).